watertap/examples/flowsheets/lsrro/tests/test_lssro_paper_analysis.py

```diff
@@ -86,18 +86,21 @@ def test_against_paper_analysis(csv_file, row_index):
         for property_name, (converter, argument) in _input_headers.items()
     }
     number_of_stages = input_arguments["number_of_stages"]
-    model, results = run_lsrro_case(
-        **input_arguments,
-        has_NaCl_solubility_limit=True,
-        permeate_quality_limit=1000e-6,
-        has_calculated_concentration_polarization=True,
-        has_calculated_ro_pressure_drop=True,
-        A_value=5 / 3.6e11,
-        B_max=None,
-        number_of_RO_finite_elements=10
-    )
-
-    if check_optimal_termination(results):
+    try:
+        model, results = run_lsrro_case(
+            **input_arguments,
+            has_NaCl_solubility_limit=True,
+            permeate_quality_limit=1000e-6,
+            has_calculated_concentration_polarization=True,
+            has_calculated_ro_pressure_drop=True,
+            A_value=5 / 3.6e11,
+            B_max=None,
+            number_of_RO_finite_elements=10
+        )
+    except ValueError:
```

Comment:
Why was this needed? To sidestep the occasional fail related to this test or something else?

Reply:
If Ipopt terminates with a weird termination condition, then Pyomo will raise a `ValueError` trying to load the results back into the model.

This simply catches this mode of failure. Note that if the solve fails (or succeeds) and we didn't expect it to the test will still fail.